app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt

```diff
@@ -145,6 +147,17 @@ object TestHelper {
     fun verifyTranslatedTextExists(text: String) =
         assertTrue(mDevice.findObject(UiSelector().text(text)).waitForExists(waitingTime))
 
+    @Suppress("Deprecation")
+    fun verifyDownloadedFileOnStorage(fileName: String) {
+        runBlocking {
+            val downloadedFile = File(
+                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS),
+                fileName
```

Comment:
I don't think we should use this in Android 10+ even with a deprecation warning, it's not best practice. Can we try `getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS)` ? I think we still need the runtime grants of `READ_EXTERNAL_STORAGE` There's also [1] talking about `MANAGE_EXTERNAL_STORAGE` 

[1] https://developer.android.com/training/data-storage/manage-all-files

Mind trying this out?

Reply:
Hi @AaronMT from my understanding `getExternalFilesDir` actually returns the path to files from our application
https://stackoverflow.com/a/10124040 so it doesn't really help in our case.

@Mugurell was kind enough to help me out with a different approach similar to what we have on AC.

I've re-ran the the test 40x on Firebase and successfully passed. ✅ 

Please take a look and share your thoughts. 😊

Reply:
MediaStore can be used on API 30+ - https://developer.android.com/training/data-storage/shared/media and the code to use it in these tests closely follows [the code used in AC for downloads](https://github.com/mozilla-mobile/android-components/blob/6dc07542819cfb29b4bf2a77f2eb8df37f1d54ea/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt#L963-L1007).

For API <=29 `getExternalStoragePublicDirectory` should be safe to use following a check for `SDK_INT`. It is deprecated starting with API 29. 
(And will be undeprecated in the next Android version. Already undeprecated in Android 12L - https://developer.android.com/sdk/api_diff/32/changes. Google..).

Reply:
If you don't mind, @Mugurell, do you have the time to review this. Everything else looks OK to me, I'm familiar with the code that And wrote in `queryDownloadMediaStore`

Reply:
Hi @AaronMT , as previously mentioned @Mugurell helped me out with this.
But sure, please go ahead and leave a review. 
Thanks!